server/src/routes/users.js

```diff
@@ -0,0 +1,28 @@
+import express from 'express'
+import { uuid } from './helpers'
+import { addUser, getUsers } from '../database'
+import { catchErrors } from '../handlers'
+
+const router = express.Router()
+
+router.post('/', catchErrors(async (req, res) => {
+  const { name } = req.body
+  const id = uuid()
+  // Will be stored as JSON in production, as string in sqlite
+  const secret = uuid(30)
+  await addUser([id, name, secret])
+  res.json({
+    id,
+    secret,
+  })
+}))
+
+
+router.get('/', catchErrors(async (req, res) => {
```

Comment:
Doesn't this endpoint expose **all** the user secret tokens???

Reply:
Why do we need an endpoint for retrieving all the users?

Reply:
removed